Charts.xcodeproj/project.pbxproj

```diff
@@ -1,1253 +1,1253 @@
 // !$*UTF8*$!
 {
-	archiveVersion = 1;
```

Comment:
the weird indent issue happens here as well. I do think we need to check. Something must be wrong.
The original file on my side is totally fine but in your commit the old file seems wrong, and you fixed with one that just the same as my old one..

Reply:
@jjatie I could reset this and just force push the valid changes. Would like me to do that? but you have to check your tab/space otherwise it will still happen

Reply:
I have a feeling that someone else's PR change them all to tabs. I haven't changed any indentation settings in years, and I've never seen an xcodeproj file with tabs. Also, why would this be an issue now when it never has before?

Reply:
I'm okay with the force push.